crates/ra_lsp_server/src/config.rs

```diff
@@ -21,6 +21,10 @@ pub struct ServerConfig {
     pub exclude_globs: Vec<String>,
 
     pub lru_capacity: Option<usize>,
+
+    /// For internal usage to make integrated tests faster.
+    #[serde(deserialize_with = "nullable_bool_true")]
+    pub with_sysroot: bool,
```

Comment:
Why not `#[cfg(test)]`?

Reply:
Integration tests build the library without `#[cfg(test)]` IIRC